src/pages/create/tabs/ReviewDeployTab/DeployProjectButton.tsx

```diff
@@ -7,6 +7,10 @@ import {
   useEditingV2FundingCycleMetadataSelector,
 } from 'hooks/AppSelector'
 import { useLaunchProjectTx } from 'hooks/v2/transactor/LaunchProjectTx'
```

Comment:
**suggestion (non-blocking)** would suggest we create a `DeployProjectWithNFTsButton` component instead.

Can be a follow-up.

Reply:
I think we should definitely do this, potentially in this PR if you're up for it.

Want to avoid risking breaking the existing project creation flow. This component has grown in complexity too much.

Reply:
Yeah very good call. I've separated everything out